CMakeMain.txt

```diff
@@ -70,7 +70,7 @@ if (APPLE)
       ${ELEMENTS_RESOURCES}
       ${ELEMENTS_APP_RESOURCES}
    )
-elseif (UNIX)
+elseif (UNIX OR (WINDOWS_GRAPHICS_LIBRARY STREQUAL "gtk"))
```

Comment:
I think this should be named WINDOWS_UI_LIBRARY, but better yet, can we have a generic one for all targets, like perhaps HOST_UI_LIBRARY, which can be "cocoa" (for MacOS), "gtk" (for Linux and Windows), and "win32" (for Windows)?

Reply:
Good idea. And Gtk also works on macOS, though cocoa works well.